tests/unit_tests/external_interaction/ispyb/test_gridscan_ispyb_store_2d.py

```diff
@@ -5,13 +5,13 @@
 from ispyb.sp.mxacquisition import MXAcquisition
```

Comment:
Actually I think one of the things that would be a fair comment on the refactoring is that I think these test modules should be rewritten so that rather than construct the inputs by essentially duplicating the logic of the callback layer (since the callback layer tests are effectively copied from here and end-to-end to the ispyb layer), the fixtures should just directly populate the raw *Info dataclasses that are passed in.

Reply:
Sure, as long as we still have the callback tests covering this then that sounds great